scripts/generate-markdown/markdown.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const fs = require('fs');
+const markdownMagic = require('markdown-magic'); // eslint-disable-line import/no-extraneous-dependencies
+
+const header = `| Terra Package      | Version | Status | Dependencies |
+|--------------------|---------|--------|--------------|
+`;
+
+const config = {
+  transforms: {
+    // see: https://github.com/camacho/markdown-magic-subpackage-list
+    SUBPACKAGELIST(content, _options, configuration) {
```

Comment:
Are do we need to pass `content`, `_options`, `configuration`? I see we use configuration below but could use
]
```diff
- path.dirname(configuration.originalPath), './packages',
+ path.join(process.cwd(), 'packages');
or
+  path.join(__dirname, .., .., 'packages');
or
```

Reply:
I dumped them all and used the first example: [ef5e90f](https://github.com/cerner/terra-clinical/pull/445/commits/ef5e90f32fb556561d92e97d2101baca9d8b08bf)